Description: 
  * Set the toolbar editable. Resolves the following bug: Learning -> Settings -> Configure Toolbars... displays an empty 'Configure Toolbars' dialog.
Author: Prathibha B <prathibhab@cdac.in>

---
Bug-Debian: https://bugs.debian.org/796200
Forwarded: no
Last-Update: 2015-09-14

Index: kbruch/src/FractionRingWidgetui.rc
===================================================================
--- kbruch.orig/src/FractionRingWidgetui.rc	2015-09-14 11:53:11.815970558 +0200
+++ kbruch/src/FractionRingWidgetui.rc	2015-09-14 11:53:11.815970558 +0200
@@ -10,7 +10,7 @@
     <Action name="file_quit"/>
   </Menu>
 </MenuBar>
-<ToolBar name="exerciseToolBar" noEdit="true" fullWidth="true">
+<ToolBar name="exerciseToolBar" noEdit="false" fullWidth="true">
   <Action name="NewTask"/>
   <Action name="Back"/>  
   <Separator/>
